api/src/main/resources/Reference_Application_Numeric_Concepts-25-2.x.xml

```diff
@@ -10,4 +10,6 @@
 <concept_numeric concept_id="5092" hi_absolute="100.0" low_absolute="0.0" units="%" allow_decimal="false"/>
 <concept_numeric concept_id="5242" hi_absolute="999.0" hi_critical="120.0" hi_normal="70.0" low_absolute="0.0" units="" allow_decimal="false"/>
 <concept_numeric concept_id="5314" low_absolute="0.0" units="cm" allow_decimal="true"/>
+<concept_numeric concept_id="165836" units="" allow_decimal="true"/>
```

Comment:
I thought you put the new concepts in Covid-19_Concepts-1.xml

Did you just accidentally change these other files?

Reply:
those are some numeric concepts for Covid-19. i still wanted to keep the backward compatibility with    versions pre 2.2

Reply:
Since you decided to separate the covid-19 in their own file, wouldn't it be consistent to also have their own concept numeric files?

Reply:
makes sense

Reply:
@dkayiwa , placed them in a separate file